fmn/sse/server.py

```diff
@@ -173,11 +175,18 @@ def write_requests(rabbit_queue):
             subscription['queue'] = rabbit_queue
 
             while True:
-                channel, method, properties, body = yield rabbit_queue.queue.get()
-                for request in subscription['requests']:
-                    msg = u'data: ' + body.decode('utf-8') + u'\r\n\r\n'
-                    request.write(msg.encode('utf-8'))
-                yield channel.basic_ack(delivery_tag=method.delivery_tag)
+                try:
+                    channel, method, properties, body = yield rabbit_queue.queue.get()
+                    for request in subscription['requests']:
+                        msg = u'data: ' + body.decode('utf-8') + u'\r\n\r\n'
+                        request.write(msg.encode('utf-8'))
+                    yield channel.basic_ack(delivery_tag=method.delivery_tag)
+                except pika_exceptions.ConsumerCancelled:
+                    _log.info(_('The server has cancelled the AMQP consumer for '
```

Comment:
I am not sure they are.

Since gettext is in stdlib, we could leave as is, but I seem to remember a time where in python-fedora we had translated error messages and that turned out to be not so helpful (for example when you can a stacktrace in a language you know nothing about).
Maybe @abadger remembers a little more the context around this, I mainly just remember we used to do it and at one point decided the pain wasn't really worth the gain.